modules/configure-ocp-oauth-identity-provider.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
```

Comment:
This module is missing the second half of the procedure.

1. Add a name for the authentication provider
2. Select a minimum access role
3. Add rules to map to roles.
![image](https://user-images.githubusercontent.com/5336227/143324268-2c075adc-cab7-43a0-94c1-5469ffa4e276.png)